src/lib/ast.rs

```diff
@@ -815,7 +819,9 @@ pub struct ReverseChildren<'a, T: Clone + 'a, U: PartialEq + Copy + 'a> {
     arena: &'a Arena<T, U>,
     node: Option<NodeId<U>>,
 }
-impl_node_iterator!(ReverseChildren, |node: &Node<T, U>| node.previous_sibling);
+impl_node_iterator!(ReverseChildren, |node: &Node<T, U>| {
```

Comment:
Let's add a blank line between the end of the function and the start of this macro call.